src/common/models/Projection.ts

```diff
@@ -450,27 +450,11 @@ export class Projection {
   }
 
   private calcCaseDensityRange(): Array<CaseDensityRange | null> {
-    return this.caseDensityByDeaths.map((caseDensityByDeaths, i) => {
-      // TODO: We were intending to calculate a low/high range for
-      // caseDensityByDeath, based on a range of CFRs, but this doesn't work when
-      // we merge with caseDensityByCases which has no range. So for now,
-      // we are punting.
-      // const caseDensityByDeathsLow = (CASE_FATALITY_RATIO / CASE_FATALITY_RATIO_UPPER) * caseDensityByDeaths;
-      // const caseDensityByDeathsHigh = (CASE_FATALITY_RATIO / CASE_FATALITY_RATIO_LOWER) * caseDensityByDeaths;
```

Comment:
Should we go ahead and remove the commented-out CASE_FATALITY_RATIO, etc. constants at the top of the file too?